tests/testthat/test-lint.R

```diff
@@ -105,7 +105,12 @@ test_that("lint() results do not depend on the position of the .lintr", {
 })
 
 test_that("lint uses linter names", {
-  expect_lint("a = 2", list(linter = "bla"), linters = list(bla = assignment_linter()), parse_settings = FALSE)
+  expect_lint(
+    "a = 2",
+    list(linter = "bla"),
+    linters = list(bla = assignment_linter()),
+    parse_settings = FALSE
+  )
```

Comment:
Unrelated change, but following the boyscout rule. This is more consistent formatting with the rest of the test suite.